tests/infra/is_snp.py

```diff
@@ -4,3 +4,10 @@
 import os
 
 IS_SNP = os.path.exists("/dev/sev")
+# TODO: Get these from the container
```

Comment:
FYI, I've added item 5. in https://github.com/microsoft/CCF/issues/4070 to cover this.

Reply:
Sounds great